config/zhimi.airpurifier.mb3.yaml

```diff
@@ -0,0 +1,280 @@
+# https://home.miot-spec.com/spec/zhimi.airpurifier.mb3
+
+external_components:
+  source: github://dhewg/esphome-miot@main
+
+esphome:
+  name: purifier
+  friendly_name: Air Purifier
+  comment: Xiaomi  Air Purifier 3/3H (zhimi.airpurifier.mb3)
+  project:
+    name: "dhewg.esphome-miot"
+    version: "zhimi.airpurifier.mb3"
+
+esp32:
+  board: esp32dev
+  framework:
+    type: esp-idf
+    sdkconfig_options:
+      CONFIG_FREERTOS_UNICORE: y
+    advanced:
+      ignore_efuse_mac_crc: true
+
+time:
+  - platform: homeassistant
+    id: homeassistant_time
+
+logger:
+  level: DEBUG
+
+api:
+  encryption:
+    key: !secret api_encryption_key
+  reboot_timeout: 0s
+  services:
+    - service: mcu_command
+      variables:
+        command: string
+      then:
+        - lambda: 'id(miot_main).queue_command(command);'
+
+ota:
+  password: !secret ota_password
+
+wifi:
+  ssid: !secret wifi_ssid
+  password: !secret wifi_password
+  ap:
+    password: !secret wifi_ap_password
+
+captive_portal:
+
+uart:
+  tx_pin: GPIO17
+  rx_pin: GPIO16
+  baud_rate: 115200
+
+miot:
+  id: miot_main
+  miot_heartbeat_siid: 13
+  miot_heartbeat_piid: 9
+
+switch:
+  - platform: "miot"
+    miot_siid: 2
+    miot_piid: 2
+    name: "Power"
+    icon: mdi:power
+  - platform: "miot"
+    miot_siid: 5
+    miot_piid: 1
+    name: "Notification Sounds"
+    icon: mdi:volume-high
+    entity_category: config
+  - platform: "miot"
+    miot_siid: 7
+    miot_piid: 1
+    name: "Child Lock"
+    icon: mdi:lock
+    entity_category: config
+
+binary_sensor:
+  - platform: "miot"
+    miot_siid: 15
+    miot_piid: 5
+    name: "Filter Door"
+    device_class: opening
+    entity_category: diagnostic
+
+select:
+  - platform: "miot"
+    miot_siid: 2
+    miot_piid: 5
+    name: "Mode"
+    options:
+      0: "Auto"
+      1: "Sleep"
+      2: "Favorite"
+      3: "None"
+  - platform: "miot"
+    miot_siid: 2
+    miot_piid: 4
+    name: "Fan Level"
+    icon: mdi:fan
+    options:
+      1: "Low"
+      2: "Medium"
+      3: "High"
+  - platform: "miot"
+    miot_siid: 6
+    miot_piid: 1
+    name: "Display Brightness"
+    icon: mdi:brightness-6
+    entity_category: config
+    options:
+      0: "High"
+      1: "Low"
+      2: "Off"
+  - platform: "miot"
+    miot_siid: 15
+    miot_piid: 12
+    name: "Temperature Display Unit"
+    icon: mdi:temperature-celsius
+    entity_category: config
+    options:
+      0: "Celsius"
+      1: "Fahrenheit"
+  - platform: "miot"
+    miot_siid: 15
+    miot_piid: 10
+    name: "Manual Level"
+    icon: mdi:speedometer
+    options:
+      1: "Level 1"
+      2: "Level 2"
+      3: "Level 3"
+
+text_sensor:
+  - platform: "miot"
+    miot_siid: 15
+    miot_piid: 13
+    miot_poll: false
+    name: "Serial Number"
```

Comment:
Does not report the serial number. The other similar model zhimi.airp.mb3a does not have this field altogether so it's better to remove this so that it's compatible with zhimi.airp.mb3a